spec/requests/posts_spec.rb

```diff
@@ -26,14 +26,30 @@
 
   # <<<<<<< CREATION POST TESTS >>>>>>
   describe 'POST /posts' do
-    it 'return http created' do
+    it 'return http created for a parent post' do
       login(user_with_posts)
       post '/api/posts', params: '{ "title": "This is a test", "content": "unit test for post creation" }',
                          headers: { 'CONTENT_TYPE' => 'application/json' }
 
       expect(response).to have_http_status(:created)
     end
 
+    it 'return http created for a child post' do
+      login(user_with_posts)
+
+      perform_enqueued_jobs do
+        post '/api/posts',
+             params: "{ \"title\": \"Child post\", \"content\": \"unit test for post creation to send emails\",
```

Comment:
You can use the to_json method the rails adds to every basic type. So you can do `{title: 'Child Post'}.to_json`